templates/terraform/custom_expand/group_name_to_id.erb

```diff
@@ -0,0 +1,3 @@
+func expandMonitoringUptimeCheckConfigResourceGroupGroupId(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
```

Comment:
If you use `<%= prefix -%><%= titlelize_property(property) -%>` here then you could name this something much more generic in case other resources need a similar one.